mllib/src/main/scala/org/apache/spark/mllib/rdd/MLPairRDDFunctions.scala

```diff
@@ -39,8 +39,8 @@ class MLPairRDDFunctions[K: ClassTag, V: ClassTag](self: RDD[(K, V)]) extends Se
    * @param ord the implicit ordering for T
    * @return an RDD that contains the top k values for each key
    */
-  def topByKey(num: Int)(implicit ord: Ordering[V]): RDD[(K, Array[V])] = {
-    self.aggregateByKey(new BoundedPriorityQueue[V](num)(ord))(
+  def topByKey(num: Int, bucketsCount: Int = 200)(implicit ord: Ordering[V]): RDD[(K, Array[V])] = {
```

Comment:
Yes, I think this breaks binary compatibility (not source compatibility) up to my knowledge.

I tested this as described in https://github.com/databricks/scala-style-guide/issues/46 but with 

```
object A {
  def toString(v: Int, extra: String = ""): String = v.toString
}
```